utils_entrypoint.py

```diff
@@ -0,0 +1,33 @@
+
+
+def load_default_args(match_config):
+    """ loads default args into the config"""
+
+    if "closest_match_variable" not in match_config:
+        match_config["closest_match_variable"] = None
+
+    if "date_exclusion_variables" not in match_config:
+        match_config["date_exclusion_variables"] = None
+
+    if "min_matches_per_case" not in match_config:
+        match_config["min_matches_per_case"] = 0
+
+    if "replace_match_index_date_with_case" not in match_config:
+        match_config["replace_match_index_date_with_case"] = None
+
+    if "output_suffix" not in match_config:
+        match_config["output_suffix"] = ""
+
+    if "indicator_variable_name" not in match_config:
+        match_config["indicator_variable_name"] = "case"
+
+    if "output_path" not in match_config:
+        match_config["output_path"] = "tests/test_output"
```

Comment:
I'm wary of testing defaults. I'd be inclined to either have no default, or make it "outputs/something"

Reply:
noted - and changed